content/about/faq.md

```diff
@@ -1,23 +1,54 @@
 ---
-title: FAQ
+title: FAQs
 weight: 1200
 type: essay
 online: false
 ---
 
-**Why is it called Quire?**
+### What is Eleventy (11ty)?
 
-From *Understanding Illuminated Manuscripts: A Guide to Technical Terms (Getty Publications)*:
-> *Quires are the “gatherings” or “booklets” of which a book is formed. Quire numeration, which began in the Late Antique period, consists of numbers written on a quire (usually on its final verso) to facilitate arrangement during binding. Quire signatures (or leaf signatures) are numbers and/or letters written in a quire to facilitate the arrangement of its internal components. These were at first ad hoc, but beginning around 1400 they might follow a system: for example, ai-aiv could be written on the first four leaves of the first quire; bi-biv on the first four leaves of the second quire, and so on. Such annotations only became widespread from the late thirteenth century on.*
+Eleventy (11ty) is a simple yet powerful [static site generator](https://quire.getty.edu/about/how-it-works/) based on JavaScript. It was created as an alternative to [Jekyll](https://jekyllrb.com/) and will replace our former static site generator, [Hugo](https://gohugo.io/). 11ty transforms a directory of various templates into HTML and is popular for its ability to mix multiple templating languages (including: HTML, Markdown, JavaScript, Liquid, Nunjucks, Handlebars, Mustache, EJS, Haml, Pug, and Custom). It is available as open source under the MIT License. Read more on the [11ty website](https://www.11ty.dev/).
```

Comment:
I'd eliminate the mention of Jekyll, I don't think it adds anything.

I'd also remove the parenthetical list of template types. While yes it supports all those, Quire will only use Liquid and JavaScript